lib/resources/example_json/r4_examples_goal.rb

```diff
@@ -76,6 +76,173 @@ module Resources
       ]
     }.freeze
 
+    R4_GOAL_REVINCLUDE_BUNDLE ||= {
+      "resourceType": 'Bundle',
+      "id": '4a1ae23d-cb80-49ac-9b3b-6abbdb7d5b92',
+      "type": 'searchset',
+      "link": [
+        {
+          "relation": 'self',
+          "url": 'https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Goal?_id=498290085&_revinclude=Provenance%3Atarget'
+        }
+      ],
+      "entry": [
+        {
+          "fullUrl": 'https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Goal/498290085',
+          "resource": {
+            "resourceType": 'Goal',
+            "id": '498290085',
+            "meta": {
+              "versionId": '1610019857000',
+              "lastUpdated": '2021-01-07T11:44:17.000Z'
+            },
+            "text": {
+              "status": 'generated',
+              "div": '<div xmlns="http://www.w3.org/1999/xhtml"><p><b>Goal</b></p><p>
+              <b>Subject</b>: Smith, Grace</p><p><b>Description</b>: Fighting depression</p>
+              <p><b>Expressed By</b>: Smith, Grace</p><p><b>Status</b>: Cancelled</p>
+              <p><b>Start Date</b>: Jan  7, 2021</p></div>'
+            },
+            "lifecycleStatus": 'cancelled',
+            "category": [
+              {
+                "coding": [
+                  {
+                    "system": 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/4003016',
```

Comment:
Update the tenant to public sandbox (C19410)
`ec2458f2-1e24-41c8-b71b-0e701af7583d`

Reply:
addressed